core/src/main/java/com/xmartlabs/bigbang/core/module/OkHttpModule.kt

```diff
@@ -73,13 +73,9 @@ open class OkHttpModule {
 
     try {
       val statFs = StatFs(dir.absolutePath)
-      val blockCount: Long
-      if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.JELLY_BEAN_MR2) {
-        blockCount = statFs.blockCountLong
-      } else {
-        @Suppress("DEPRECATION")
-        blockCount = statFs.blockCount.toLong()
-      }
+      @Suppress("DEPRECATION")
+      val blockCount = if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.JELLY_BEAN_MR2)
```

Comment:
I disagree. Extracting a condition to a private function actually makes things clearer. I do agree that we could have generic function in another place, but as a general rule, making this kind of change is ok to me